daml-lf/spec/daml-lf-2.rst

```diff
@@ -4545,18 +4545,14 @@ ordered by keys according to the comparison function ``LESS``.
 
   Returns an empty generic map.
 
-  [*Available in versions >= 1.11*]
-
 * ``GENMAP_INSERT : ∀ α. ∀ β.  α → β → 'GenMap' α β → 'GenMap' α β``
 
   Inserts a new key and value in the map. If the key is already
   present according the builtin function ``EQUAL``, the associated
   value is replaced with the supplied value, otherwise the key/value
-  is inserted in order according to the builtin function ``LESS`` applied
-  on keys. This raises a runtime error if it tries to compare
-  incomparable values.
-
-  [*Available in versions >= 1.11*]
+  is inserted in order according to the builtin function ``LESS``
+  applied on keys. This raises a runtime error if it the first
+  argument is not a comparable value.
```

Comment:
```suggestion
  applied on keys. This raises a runtime error if the first
  argument is not a comparable value.
```

Reply:
there's also a couple of redundant `it`s in the sections below but github doesn't allow `suggestion`s on untouched lines

Reply:
Good catch ! Thanks.